docs/ldml/tr35-general.md

```diff
@@ -951,15 +951,17 @@ Some of the constraints reference data from the unitIdComponents in [Unit_Conver
     </ul></td></tr>
 
 <tr><td><a name='unit_constant' href='unit_constant'>unit_constant</a></td><td>:=</td>
-    <td>("1"[0-9]+ | [2-9][0-9]*)("e" ("1"[0-9]+ | [2-9][0-9]*))?
+    <td>[1-9][0-9]* ("e" [1-9][0-9]*)?
```

Comment:
this will allow "1" in "per-1" . Is that ok?

Reply:
That is disallowed by the constraint below.